.github/workflows/lint.yaml

```diff
@@ -5,7 +5,10 @@ jobs:
     eslint:
         name: ESLint
         runs-on: ubuntu-latest
-
+        env:
+            # We don't want Puppeteer to automatically download a browser when dependencies are being installed
+            PUPPETEER_SKIP_DOWNLOAD: 'false'
```

Comment:
`skip_download: false` makes it _not_ download the browser ? Are you sure about that ?

Reply:
Ouch, the fatigue got me, thanks!